strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/users/UserController.java

```diff
@@ -239,6 +239,11 @@ public ResponseEntity delete(@ApiParam(value = "The name of the user") @PathVari
             return getFailedResponseEntity(HttpStatus.FORBIDDEN, OWN_USER_DELETE_FORBIDDEN, accept);
         }
 
+        if (StringUtils.equals("admin", username))
```

Comment:
@anki2189 why this change needed for?
@carlspring @steve-todorov do we have such requirments to prevent "admin" user been deleted?

Reply:
There was one disabled TestCase (`UserControllerTestIT.testDeletingRootAdminShouldBeForbidden`) for deleting admin User . 
According to test case deletion of root admin should not be allowed , so added this code in UserController.

Reply:
ok, thanks for clarifying